public class temperature {

    static class Temperature {
        double celsius;

        Temperature(double celsius) {
            this.celsius = celsius;
        }

        void display() {
            double fahrenheit = (celsius * 9 / 5) + 32;

            System.out.println("Temperature in Celsius: " + celsius);
            System.out.println("Temperature in Fahrenheit: " + fahrenheit);
        }
    }

    public static void main(String[] args) {
        Temperature t1 = new Temperature(30);

        t1.display();
    }
}
